cmd/cycloid/common/flags.go

```diff
@@ -34,6 +34,13 @@ func WithFlagCan(cmd *cobra.Command) string {
 	return flagName
 }
 
+func WithPersistentFlagCan(cmd *cobra.Command) string {
```

Comment:
Why not use method WithFlagCan instead of creating a new one? Do we really need a persistenFlag for this case?

Reply:
yep, 1 `create cred <type>` is a sub command of 2 `create cred` so if we want a flag from 2 available on 1, it have to be a persistent one

I think ideally we should do a full refactor to change all With* function by adding a new persistent bool to factorize all of them. But it might need a dedicated work/PR